### DISCUSSION TITLE
Urgent Help Needed: Github Account Flagged and Unresponsive, Seeking Resolution

### DISCUSSION THREAD

**Discussion by 03prashantk** [community/community#118774] [Discussions]
### Select Topic Area

Question

### Body

Hey Github folks, 

My Github account (@03prashantpk) is flagged, and I'm reaching out for assistance. I've already raised this issue via my main account (@03prashantpk), but I'm also starting this discussion from this alternate account in case I can't reach the audience with the other one.

So, my secondary account was accessed from an unauthorized source, leading to some concerning activity on my profile. Three new repositories were added without my consent, prompting Github to take security measures and lock my account. Understandably, I'm unable to access my repositories, and despite reaching out through various channels like Twitter and LinkedIn, as well as creating a support ticket, I've received no resolution after 16 days.

I've ensured that all possible security measures, including 2FA, are in place, yet I remain locked out of my account. With 71 repositories, including vital projects for job placement and freelancing, this situation is becoming increasingly critical for me.

Any assistance or insight on how to resolve this issue would be greatly appreciated. Thank you for your attention.

It's incredibly disheartening and frustrating that despite reaching out to Github for the past seven days, I haven't received any response. What's more, I'm a pro user, which adds to the disappointment. As users, we entrust Github with our code, but it's alarming to realize that there's a possibility of them being unresponsive or even turning against us. The lack of proper contact methods exacerbates the issue; the only available option seems to be through tickets, which unfortunately haven't yielded any response either.
![githubProblem](https://github.com/community/community/assets/165763830/53cea703-3be3-4f3a-ab26-acf46e2a0edf)

**Comment by LiteBrite82** [comment]
Hi @03prashantk 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by 03prashantpk** [reply]
Could you please help me reopen this discussion and assist in finding a way to escalate it on GitHub? It's been 18 days without a response, and urgency is a concern. Additionally, it's disconcerting that despite trusting them with our code, they haven't addressed our concerns. It would be greatly appreciated if the community could offer support in this situation.


**Once again it's really important** 
Account: @03prashantpk

Please help: @LiteBrite82 @mgriffin @zackfern @elisealix22 @Bestra @samus-aran @timoyates @keisaacson @turbo @Nishnha @masachs @omerbensaadon @thejoebourneidentity @tanmayeekamath @robertnopickle @Steve-Glass @natalierjackson @realkimpossible @rileybroughten @grandmas-favorite @jmeridth @martinwoodward @davidstaheli @otter-computer @Iron-Ham @jeffbeehler @bailey @ossanna16 @talum 

As longstanding users, I believe you have influence in the community. Could you please help by forwarding this issue, perhaps on Twitter or another platform? It would greatly assist me as I'm feeling quite helpless. I manage 71 repositories, including client code, and rely on GitHub for backups. But what if they flag us without reason? Your support would mean a lot in this situation.
Account: @03prashantpk or https://www.google.com/search?q=enally+prashant You can see it here.
